recipe/meta.yaml

```diff
@@ -4,23 +4,21 @@
 {% set version = "8.0.121" %}
 {% set zulu_build = "8.20.0.5" %}
 {% set build_number = 0 %}
-{% set md5_linux = "e5f4b1d997e50ffe4998c68c8ec45403" %}
-{% set md5_osx = "87cbe3789cba2851bd3a93c28a1b1672" %}
-{% set md5_win = "89ea67c5966a44627809d9a46a2f3b25" %}
+{% set sha256_linux = "7fdfb17d890406470b2303d749d3138e7f353749e67a0a22f542e1ab3e482745" %}
+{% set sha256_osx = "2a58bd1d9b0cbf0b3d8d1bcdd117c407e3d5a0ec01e2f53565c9bec5cf9ea78b" %}
+{% set sha256_win = "35414df28f17704546b9559b5e62b4d00cdc8fdfd349116be4f987abaeaaff68" %}
```

Comment:
I downloaded them via `https` and got the new values. This is therefore as secure as the `https` server that Azul runs. I also verified that the `md5` matches so this is as secure as it was previously.
`md5` over` http` is by my calculation much less secure than making this change, regardless of what Azul list on their website.